### DISCUSSION TITLE
Use of the wavelog for the club station.

### DISCUSSION THREAD

**Discussion by miklobit** [wavelog/wavelog#1416] [Q&A] [answered]
Is it possible to use the wavelog at the club station so that qso's logged in from different accounts can be seen together? 
As i see is possible to assign the same callsign to many users, but I do not see anywhere a common log view for all qso's under the same callsign, regardless of which user logged it.

**Comment by HB9HIL** [comment, marked as answer]
Working on it

https://github.com/wavelog/wavelog/pull/1334

**Reply by HB9HIL** [reply]
PS: Happy New Year :)
